### DISCUSSION TITLE
Is there any way to pass `security` to `KafkaRouter`?

### DISCUSSION THREAD

**Discussion by gegnew** [ag2ai/faststream#951] [Q&A]
It appears that you can only pass it to `KafkaBroker`, which makes integration with FastApi difficult.

**Comment by Lancetnik** [comment]
@gegnew `KafkaRouter(security=...)` is a valid case too
What problem you have?

**Comment by Lancetnik** [comment]
Also, I created a related Issue #957

**Reply by Lancetnik** [reply]
Yep, all works fine https://github.com/airtai/faststream/pull/958
